src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskForSearchCompletionControllerTest.java

```diff
@@ -51,9 +51,10 @@ public void should_return_200_with_appropriate_task_to_complete() {
         Stream<CompletableTaskScenario> scenarios = tasksToCompleteScenarios();
         scenarios.forEach(scenario -> {
 
-            TestVariables testVariables = common.setupWATaskAndRetrieveIds();
+            TestVariables testVariables = common.setupWATaskAndRetrieveIds("requests/ccd/wa_case_data.json");
             initiateTask(caseworkerCredentials.getHeaders(), testVariables,
-                "processApplication", "process application", "process task");
+                         "processApplication", "process application", "process task"
+            );
 
```

Comment:
searchCompletion has no scenarios showing nextHearingDate as having value. All null tests. Can we include where the nextHearingDate is NotNull?